src/app/legacy/containers/CpsRelatedContent/index.test.jsx

```diff
@@ -85,44 +87,6 @@ describe('CpsRelatedContent', () => {
     expect(document.querySelectorAll(`[role='region']`).length).toBe(1);
   });
 
-  it('should render timestamps in milliseconds when page data has timestamps in seconds', async () => {
```

Comment:
I could not get this test passing locally, despite https://www.epochconverter.com/ telling me that a timestamp of `1234567890` = 13 February 2009... Is this acceptable?

<img width="634" alt="image" src="https://github.com/bbc/simorgh/assets/58214768/e2698931-e973-4ef0-a494-8705028d6790">